gradle.properties

```diff
@@ -6,4 +6,4 @@ deployUrl = https://oss.sonatype.org/service/local/staging/deploy/maven2/
 # Enable the daemon by adding org.gradle.daemon in USER_HOME/.gradle/gradle.properties
 org.gradle.parallel=true
 
-java6Home=/Library/Java/JavaVirtualMachines/1.6.0_65-b14-462.jdk/Contents/Home
+java6Home=/Library/Java/JavaVirtualMachines/1.6.0.jdk/Contents/Home
```

Comment:
None of the other properties are environment dependent, they are required by the build and stable across all use. The `SONATYPE*` ones are an exception but must exist for the build to work from what I can remember.